server/helpers/planner_helper/planner_helper_data_types.py

```diff
@@ -68,19 +70,12 @@ class LandmarkCategory(Enum):
 
 class ChoiceInfo(BaseModel):
     landmark: Optional[Landmark] = None  # landmark
-    max_num_plans: Optional[
-        int
-    ] = 0  # the maximum number of plans included in a first achiever
-    action_name_plan_idx_map: Optional[
-        Dict[str, List[int]]
-    ] = (
-        dict()
-    )  # keys are first-achievers (or edges) available for the next choice
-    action_name_plan_hash_map: Optional[
-        Dict[str, List[str]]
-    ] = (
-        dict()
-    )  # keys are first-achievers (or edges) available for the next choice
+    # the maximum number of plans included in a first achiever
```

Comment:
@jkeskingvillage headsup, i have killed the optionals, since there is a default value attached anyway